src/node_snapshotable.cc

```diff
@@ -1151,16 +1151,20 @@ ExitCode SnapshotBuilder::Generate(SnapshotData* out,
       Context::Scope context_scope(main_context);
 
       // Create the environment.
-      env = new Environment(main_instance->isolate_data(),
-                            main_context,
-                            args,
-                            exec_args,
-                            nullptr,
-                            node::EnvironmentFlags::kDefaultFlags,
-                            {});
-
-      // Run scripts in lib/internal/bootstrap/
-      if (env->principal_realm()->RunBootstrapping().IsEmpty()) {
+      uint64_t env_flags = EnvironmentFlags::kDefaultFlags |
+                           EnvironmentFlags::kInspectorOnlyAfterBootstrap;
+      if (snapshot_type != SnapshotMetadata::Type::kFullyCustomized) {
```

Comment:
```suggestion
      // It's not guaranteed that a context that goes through
      // v8_inspector::V8Inspector::contextCreated() is runtime-independent,
      // so do not start the inspector on the main context when building
      // the default snapshot.
      if (snapshot_type == SnapshotMetadata::Type::kDefault) {
```

Reply:
Done in b634c382bbf6060273d95ccb96b1804211fbe8e6 :+1: